emulate-browsers/chrome-80/index.ts

```diff
@@ -104,6 +104,22 @@ export default class Chrome80 {
     return this.domOverrides.build();
   }
 
+  public async newWorkerInjectedScripts() {
+    const result = this.domOverrides.build([
+      'Error.captureStackTrace',
+      'Error.constructor',
+      'navigator.deviceMemory',
+      'navigator',
+      'MediaDevices.prototype.enumerateDevices',
+      'Notification.permission',
+      'Permission.prototype.query',
+      'WebGLRenderingContext.prototype.getParameter',
+      'HTMLMediaElement.prototype.canPlayType',
+      'RTCRtpSender.getCapabilities',
+    ]);
+    return result;
+  }
+
```

Comment:
I'm not sure how we should handle changes to Emulators. This file is auto-generated by Slab. I'm not sure if it should be directly modified here.